code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -27,11 +27,11 @@ LINEN BINS
 	if(I.is_sharp())
 		cut_time = 60 / I.sharpness
 	if(cut_time)
-		to_chat(user, "<span  class='notice'>You begin cutting the [src].</span>")
+		to_chat(user, "<span  class='notice'>You begin cutting up [src].</span>")
```

Comment:
best practice is to include \the before [src] that way if it gets renamed in adminbus it won't mess with the grammar; not a major issue